### PR TITLE
Use instance role for cyhy-feeds instead of explicit user creds

### PR DESCRIPTION
**Pull request by jsf9k**
In this pull request I adjust the AWS infrastructure to allow the use of an instance role for running cyhy-feeds on the Mongo instance.

This pull request goes along with cisagov/cyhy-feeds#19.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
Excelsior! 💡

**Comment by jsf9k** [comment]
Since @KyleEvers is the lead on cyhy-feeds, I'm going to wait until he is able to review and  approve before merging.

**Comment by KyleEvers** [comment]
Looks great 🥇

**Pull request merged**
